### PR TITLE
Improve markdown text extraction when chunking

### PR DESCRIPTION
**Pull request by mhham**
In a nutshell, we call `pypandoc.convert_text` on the `current_text` in our custom markdown text chunks extraction function.

### CODE REVIEW

File: python-lib/text_extraction/__init__.py [hunk 1]
```diff
@@ -57,57 +58,57 @@ def extract_text_content(file_bytes, extension, with_pandoc):
...
+                        'metadata': "Page {}".format(page_id + 1) if metadata_as_plain_text else {"page": page_id + 1},
+                        'error_message': ""
+                    }
+                    for page_id, page in enumerate(pdf_pages)
+                ]
+            else:
+                return _extract_pdf_chunks(filename, pdf_pages, outline, metadata_as_plain_text)
+        elif extension == "md":
+            if not with_pandoc:
+                raise ValueError("pandoc is required to extract chunks from markdown files.")
```
Comment (StanislasGuinel) [inline python-lib/text_extraction/__init__.py:84]: should we make it work for "md" file without pandoc ? Using the `pypandoc.convert_text` method depending on an args of the `_extract_markdown_chunks` method ?

**Review by StanislasGuinel — COMMENTED** [review]
Just a small comment regarding the restriction to md files.
Otherwise LGTM

File: python-lib/text_extraction/__init__.py (hunk 1 above)
Comment (mhham) [reply python-lib/text_extraction/__init__.py]: https://github.com/dataiku/dss-plugin-tesseract-ocr/pull/76/commits/50c1285f57609a404f941270c11deacfd1835cd8

**Review by StanislasGuinel — APPROVED** [review]
LGTM

**Pull request merged**